test.js

```diff
@@ -35,6 +35,21 @@ test('bootstrapper - opts', async function (t) {
   await node.destroy()
 })
 
+test('bootstrapper - opts.bootstrap', async function (t) {
+  const node1 = DHT.bootstrapper(49737, '127.0.0.1')
```

Comment:
this needs to pick a random free port instead, otherwise it'll clash with other apps

do this

``` js
async function freePort () {
  const udx = new UDX()
  const sock = udx.createSocket()
  sock.bind(0)
  const port = sock.address().port
  await sock.close()
  return port
}
```

Reply:
I guess I will use this one as it's already there:
https://github.com/mafintosh/dht-rpc/blob/949d146db4cb79bb6baf8ca08716b141bc79f060/test.js#L388

And I will update the other bootstrapper tests to also use random ports, if that's okay